shared/src/business/utilities/pdfGenerator/documentTemplates/Order.jsx

```diff
@@ -24,7 +24,7 @@ export const Order = ({ options, orderContent, orderTitle, signatureText }) => {
             Clerk of the Court
           </p>
         </div>
-      )}
-    </>
+      )}{' '}
```

Comment:
Does this just add a space at the end?

Reply:
it doesn't seem like we needed this. I'll remove it in the new pr